thunderdomebot.py

```diff
@@ -17,11 +17,14 @@
 
 @bot.event
 async def on_ready():
-    print("Discord bot online!")
+    print("ThunderDomeBot online!")
+    await bot.change_presence(activity=discord.Game(name="Restarting..."), status=discord.Status.dnd)
```

Comment:
It does not switch to do not disturb and I'm not sure why... Maybe something about the game activity?